src/main/scala/Joint.scala

```diff
@@ -62,9 +131,14 @@ case class Joint(normalVec: (Double, Double, Double), localOrigin: (Double, Doub
     * vector for the plane, and the second item is the distance of the plane from the origin.
     */
   def globalCoordinates: Seq[((Double, Double, Double), Double)] = {
-    val Nplane = DenseVector[Double](a, b, c)
-    val strike = (dipDirection - math.Pi / 2) % (2 * math.Pi) // CHECKED: strike = dipDirection - pi/2 (US convention)
-    val Nstrike = DenseVector[Double](math.cos(strike), math.sin(strike), 0.0)
+    val normalDirection = if (c > -Joint.EPSILON) {
+      -1.0
+    } else {
+      1.0
+    }
+    val Nplane = normalDirection * DenseVector[Double](a, b, c) // Ensures that normal will always point in -z global direction (ensures right-handed local coordinates)
```

Comment:
You know, it might actually be clearer to combine the above `normalDirection` calculation with this step here like so:
```scala
val Nplane = if (c > -Joint.EPSILON) {
    -1.0 * DenseVector[Double](a,b,c)
} else {
    DenseVector[Double](a,b,c)
}
````
And then you can put the comment about the -z global direction here. That way, all of the logic for this is concentrated in one place and has a helpful comment with it.